include/m_common_data.h

```diff
@@ -159,86 +150,90 @@ typedef struct FamicomEmuCommonData {
     /* 0x20 */ s16 unk20;
     /* 0x22 */ s16 unk22;
     /* 0x24 */ s16 unk24;
-}FamicomEmuCommonData; // size >= 0x26
+} FamicomEmuCommonData; // size >= 0x26
+
+typedef void (*CommonData_100E4_unk_00)(struct Game_Play* game_play);
 
-typedef void (*CommonData_100E4_Func)(struct Game_Play*);
+typedef struct CommonData_unk_100E4 {
+    /* 0x00 */ CommonData_100E4_unk_00 unk_00;
+} CommonData_unk_100E4; // size >= 0x4
 
 typedef struct CommonData {
-    /* 0x00000 */ u8 unk00000[0x14];
+    /* 0x00000 */ u8 unk_00000[0x14];
     /* 0x00014 */ s32 unk_00014;
     /* 0x00018 */ u8 nowNpcMax;
     /* 0x00019 */ u8 removeAnimalIdx;
-    /* 0x0001A */ u8 unk1A[0x20 - 0x1A];
-    /* 0x00020 */ Private_c private[PLAYER_NUM]; /* player data */
-    /* 0x02F60 */ mLd_land_info_c land_info; /* town name & id */
-    /* 0x02F6A */ u8 unk02F6A[0x61E];
+    /* 0x0001A */ u8 unk_1A[0x20 - 0x1A];
+    /* 0x00020 */ PrivateInfo saveFilePrivateInfo[PLAYER_NUM]; // player data
+    /* 0x02F60 */ LandInfo landInfo; // town name & id
+    /* 0x02F6A */ u8 unk_02F6A[0x61E];
     /* 0x03588 */ mHm_hs_c homes[PLAYER_NUM];
-    /* 0x062A8 */ mFM_fg_c fg[FG_BLOCK_Z_NUM][FG_BLOCK_X_NUM]; /* fg items (fg = foreground?) */
-    /* 0x09EA8 */ u8 unk09EA8[0x70];
-    /* 0x09F18 */ Animal_c animals[ANIMAL_NUM_MAX]; /* villagers in town */
-    /* 0x0EC70 */ u8 unk0EC70[0x134];
+    /* 0x062A8 */ mFM_fg_c fg[FG_BLOCK_Z_NUM][FG_BLOCK_X_NUM]; // fg items (fg = foreground?)
```

Comment:
This struct name doesn't seem to appear anywhere in symbols, so it's probably better to reaname it.
I believe I have it as `FMForeground` in my npc scratch